### ISSUE TITLE
Server requirements check

### ISSUE THREAD

**Issue by maikschneider** [xima-media/xima-deployer-extended-typo3#11]
This deployment configuration assumes a couple of settings that need to be done before doing a deployment. It would be nice to have an automated test of these server requirements. The PR #6 by @martipoe already contains some of these checks.

**Comment by maikschneider** [comment]
Done in [v1.5.0](https://github.com/xima-media/xima-deployer-extended-typo3/releases/tag/v1.5.1). Thanks @martipoe!
